mealie/services/scraper/cleaner.py

```diff
@@ -412,6 +414,10 @@ def clean_time(time_entry: str | timedelta | None, translator: Translator) -> No
         return None
 
     match time_entry:
+        case numbers.Number():
```

Comment:
You mentioned this in a comment, but can we just make this `int | float`? If not this is fine as-is

Reply:
I dont think its possible to match 2 types in a clean way, where as Numbers does provide that here(it also allows for expansion, even though i think `int` and `float` are the only 2 numeric types that will be passed in)